# request
bool enable
---
# response
bool state